.travis.yml

```
@@ -139,19 +139,28 @@ jobs:
       # Workaround for bug in libopus's opus.h including <opus_multistream.h>
       # instead of <opus/opus_multistream.h>.
       # Virtual X (Xvfb) is needed for analyzer waveform tests
+      addons:
+        homebrew:
+          update: true
+          packages:
+            - ccache
       env: >-
-        CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=OFF -DCMAKE_PREFIX_PATH=/usr/local/opt/qt5/"
+        MIXXX_ENVNAME=2.3-j00004-497fe02e-osx10.11-x86_64-release
+        MIXXX_ENVPATH="${HOME}/buildenv"
+        CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=OFF -DFAAD=OFF -DWAVPACK=OFF -DWAVPACK=OFF -DMAD=OFF -DMODPLUG=OFF -DCMAKE_PREFIX_PATH=${MIXXX_ENVPATH} -DQt5_DIR=${MIXXX_ENVPATH}/Qt-5.12.3/lib/cmake/Qt5/"
```

Do we have to hardcode the Qt version here? Would `*` work?
```suggestion
        CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=OFF -DFAAD=OFF -DWAVPACK=OFF -DWAVPACK=OFF -DMAD=OFF -DMODPLUG=OFF -DCMAKE_PREFIX_PATH=${MIXXX_ENVPATH} -DQt5_DIR=${MIXXX_ENVPATH}/Qt-*/lib/cmake/Qt5/"
```

---

That was my initial attempt but didn't work for some reason. I can try again.

---

Does not work unfortunately: https://travis-ci.org/github/Holzhaus/mixxx/jobs/687880337#L14832

---

How do you know the expression evaluated correctly? It is not echoed anywhere in the log.

---

If you take the current state of this PR and replace the Qt version with a glob it stops working.